pkg/aws/client.go

```diff
@@ -48,25 +48,39 @@ func New(o Options) *Client {
 
 //ListRecordSets retrieve all records existing in the specified hosted zone
 func (c *Client) ListRecordSets(zoneID string) ([]*route53.ResourceRecordSet, error) {
+	records := make([]*route53.ResourceRecordSet, 0)
+
 	client, err := c.initRoute53Client()
 	if err != nil {
 		return nil, err
 	}
 
-	// TODO: implement paging
 	params := &route53.ListResourceRecordSetsInput{
 		HostedZoneId: aws.String(zoneID),
 	}
-	rsp, err := client.ListResourceRecordSets(params)
-	if err != nil {
-		return nil, err
-	}
 
-	if rsp == nil {
-		return nil, ErrInvalidAWSResponse
-	}
+	for {
+		rsp, err := client.ListResourceRecordSets(params)
+		if err != nil {
+			return nil, err
+		}
 
-	return rsp.ResourceRecordSets, nil
+		if rsp == nil {
+			log.Warnln("Empty response from AWS ListRecordSets API")
+			break
+		}
+
+		records = append(records, rsp.ResourceRecordSets...)
```

Comment:
i think you can use [this helper method](https://godoc.org/github.com/aws/aws-sdk-go/service/route53#Route53.ListResourceRecordSetsPages) to do the pagination.